TWLight/emails/templates/emails/project_page_2021_email-subject.html

```diff
@@ -0,0 +1,2 @@
+{% comment %} English-only, do not translate.{% endcomment %}
```

Comment:
Same here

Reply:
This gets loaded via [djmail's templatemail](https://bameda.github.io/djmail/#_templatemail)